### PR TITLE
DevDB 23Q4 continued tweaks

### PR DESCRIPTION
**Pull request by fvankrieken**
Tweaks to get new census data loading, add a space so that sqlite statement is "valid"

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -54,6 +54,10 @@ on:
           - WARNING
           - INFO
           - DEBUG
+      build_name:
+        description: "Build name (defaults to branch name)"
+        type: string
+        required: false
```
Comment (fvankrieken) [inline .github/workflows/build.yml:60]: Figured it's nice to have the option to specify a custom build name

Comment (damonmcc) [reply .github/workflows/build.yml:60]: I like it. I think any DB schema that isn't the same as an existing repo branch will get deleted at the end of the week though, just noting

**Review by damonmcc — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (fvankrieken) [reply .github/workflows/build.yml:60]: Yeah it's a little bit of a downside. But still nice every once in a while to be able to run two builds from the same branch without much hassle

**Comment by fvankrieken** [comment]
Tmp fix with the space in the yml template while awaiting official fix of https://github.com/OSGeo/gdal/issues/9093

**Pull request merged**